CMakeLists.txt

```diff
@@ -9,6 +9,9 @@ include(CTest)
 set(REQUIRED_BOOST_VERSION 1.58)
 find_package(Boost ${REQUIRED_BOOST_VERSION} REQUIRED)
 
+option(CPP_BINDGEN_GT_LEGACY OFF)
```

Comment:
You must pass a doc-string to option (in this case, OFF is the doc string, and value defaults to OFF)

Reply:
Thanks! I always make this mistake...